.github/workflows/autoclose.yml

```diff
@@ -0,0 +1,21 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
```

Comment:
just copied the template, is there a difference if we decide on a different time? Taking into account the action acts on issues that spawn days (30, 60) a couple of hours shouldn't make a difference so any time should be ok? Lmk if I should rethink this

Reply:
Once a day makes sense to me! I was just curious why 01:30 :) Maybe there's a "fun time" we can have it a run at like ... 0:42 or similar?

Reply:
Why there is no option to make close this when new version is released and ping issuer if it is still problem?

Checking every 30 days if something still work I have better idea to spend time... Or when ping or related UNIX network shit is forbidden by firewall and network operators and throw out window by hw manufacturers and replaced by nice microservice app :) Network is mess by design and so is some Wine/Proton establishment :)

Reply:
There are new versions of the client and new versions of the snap. We don't manage steam updates, those are handled automatically by the client and those could be bug fixes. On the other side we do manage snap updates which are far less frequent. Since we don't have control over steam client updates, we don't know exactly what they're fixing and the issues may still be there